rules/microsoft-365/impact_microsoft_365_unusual_volume_of_file_deletion.toml

```diff
@@ -0,0 +1,46 @@
+[metadata]
+creation_date = "2021/07/15"
+maturity = "development"
+updated_date = "2021/07/15"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies when Microsoft Cloud App Security reported a user has deleted a unusual a large volume of files.
+"""
+false_positives = ["Unknown"]
```

Comment:
Users cleaning old documents or system administrators cleaning them after migrating could be some ideas here